src/main/java/mate/academy/rickandmorty/controller/CharacterController.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.rickandmorty.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.internal.InternalCharacterDto;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Character management", description = "Endpoints for managing products")
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/character")
+public class CharacterController {
+    private final CharacterService characterService;
+
+    @GetMapping("/random")
+    @Operation(summary = "Get randomly character", description = "Return a character by random id")
```

Comment:
```suggestion
    @Operation(summary = "Get random character", description = "Return a character by random id")
```
Also `Return a character by random id` I don't see any id in method declaration, think of better description, it's confusing